docs/partials/_remix-deploy-example.mdx

```diff
@@ -36,7 +36,156 @@ Please double-check that the compiler version of the Remix IDE matches the compi
+
+To start testing, create a new folder called `tests` at the root level of your Remix workspace.
+This is the default directory for tests.
+If you want to change the test directory, you can do so by clicking on the double-check icon and specifying the path in the `Test directory` field.
+
+Next, open the Solidity file which you want to test, then switch to the "Solidity Unit Testing" plugin and click on the `Generate` button.
+In the test directory, a test file will be created for your selected file.
+This file contains information about developing tests for a contract.
+
+Below is an example of a generic test file `myNFT_test.sol`:
+
+```solidity
+// SPDX-License-Identifier: GPL-3.0
+        
+pragma solidity >=0.4.22 <0.9.0;
+
+// This import is automatically injected by Remix
+import "remix_tests.sol"; 
+
+// This import is required to use custom transaction context
+// Although it may fail compilation in 'Solidity Compiler' plugin
+// But it will work fine in 'Solidity Unit Testing' plugin
+import "remix_accounts.sol";
+import "../contracts/MyNFT.sol";
+
+// File name has to end with '_test.sol', this file can contain more than one testSuite contracts
+contract testSuite {
+
+    /// 'beforeAll' runs before all other tests
+    /// More special functions are: 'beforeEach', 'beforeAll', 'afterEach' & 'afterAll'
+    function beforeAll() public {
+        // <instantiate contract>
+        Assert.equal(uint(1), uint(1), "1 should be equal to 1");
+    }
+
+    function checkSuccess() public {
+        // Use 'Assert' methods: https://remix-ide.readthedocs.io/en/latest/assert_library.html
+        Assert.ok(2 == 2, 'should be true');
+        Assert.greaterThan(uint(2), uint(1), "2 should be greater than to 1");
+        Assert.lesserThan(uint(2), uint(3), "2 should be lesser than to 3");
+    }
+
+    function checkSuccess2() public pure returns (bool) {
+        // Use the return value (true or false) to test the contract
+        return true;
+    }
+    
+    function checkFailure() public {
+        Assert.notEqual(uint(1), uint(1), "1 should not be equal to 1");
+    }
+
+    /// Custom Transaction Context: https://remix-ide.readthedocs.io/en/latest/unittesting.html#customization
+    /// #sender: account-1
+    /// #value: 100
+    function checkSenderAndValue() public payable {
+        // account index varies 0-9, value is in wei
+        Assert.equal(msg.sender, TestsAccounts.getAccount(1), "Invalid sender");
+        Assert.equal(msg.value, 100, "Invalid value");
+    }
+}
+```
+
+Take the auto-generated template as a basis to create sufficient unit tests to ensure that your contract works as expected under different scenarios.
+Remix injects a built-in assert library for testing.
+You can visit the library documentation [here](https://remix-ide.readthedocs.io/en/latest/assert_library).
```

Comment:
URL returns 404. Also, instead of **here**, please use explicit heading text. 

<img width="626" alt="image" src="https://github.com/user-attachments/assets/f2bcf018-e355-47d2-b396-9c49e24309be" />